### ISSUE TITLE
[FE] New common component for Link

### ISSUE THREAD

**Issue by VishukumarU** [Vishu-Organization/FlavorHive#106]
Create a common component to display links. This should have all the implementations for accessibility and widths

![Image](https://github.com/user-attachments/assets/2a2cc224-a019-403e-8e31-f38eb8d4bce5)
